types/react-toggle/index.d.ts

```diff
@@ -1,17 +1,17 @@
-// Type definitions for react-toggle 2.2
+// Type definitions for react-toggle 4.0
 // Project: https://github.com/aaronshaf/react-toggle
 // Definitions by: Karol Janyst <https://github.com/LKay>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import { Component, HTMLAttributes, ReactNode } from "react";
+import { Component, InputHTMLAttributes, ReactNode } from "react";
 
 export interface ToggleIcons {
     checked?: ReactNode;
     unchecked?: ReactNode;
 }
 
-export interface ToggleProps extends HTMLAttributes<any> {
+export interface ToggleProps extends InputHTMLAttributes<HTMLInputElement> {
     "aria-labelledby"?: string;
     "aria-label"?: string;
```

Comment:
It think `aria-` props are special-cased by the parser to always be allowed as string props...... but I'm not sure on this. I know that `data-` is.